packages/api-derive/src/balances/all.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017-2019 @polkadot/api-derive authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { combineLatest, of, Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { ApiInterface$Rx } from '@polkadot/api/types';
+import { AccountId, AccountIndex, Address, Balance, BalanceLock, BlockNumber } from '@polkadot/types';
+import { bnMax } from '@polkadot/util';
+
+import { idAndIndex } from '../accounts/idAndIndex';
+import { DerivedBalances } from '../types';
+import { drr } from '../util/drr';
+
+const EMPTY_ACCOUNT = new AccountId(new Uint8Array(32));
+
+export function all (api: ApiInterface$Rx) {
```

Comment:
Would like an e2e test here so you can validate it actually does what it says on the tin - https://github.com/polkadot-js/api/blob/master/packages/api/test/e2e/promise-queries.spec.js